samples/highcharts/accessibility/accessible-pattern-fills/demo.js

```diff
@@ -0,0 +1,396 @@
+        borderColor: '#8ae'
+    },
+
+    plotOptions: {
+        series: {
+            label: {
+                enabled: false
+            },
+            dataLabels: {
+                enabled: false,
+                connectorColor: '#777',
+                format: '<b>{point.name}</b>: {point.percentage:.1f} %'
+            },
+            point: {
+                events: {
+                    click: function () {
+                        window.location.href = this.website;
+                    }
+                }
+            },
+            cursor: 'pointer',
+            borderWidth: 3
+        }
+    },
+
+    series: [{
+        name: 'Screen reader usage',
+        data: [{
+            name: 'NVDA',
+            y: 40.6,
+            color: getColorPattern(0),
+            website: 'https://www.nvaccess.org',
+            accessibility: {
+                description: 'This is the most used desktop screen reader'
+            }
+        }, {
+            name: 'JAWS',
+            y: 40.1,
+            color: getColorPattern(1),
+            website: 'https://www.freedomscientific.com/Products/Blindness/JAWS'
+        }, {
+            name: 'VoiceOver',
+            y: 12.9,
+            color: getColorPattern(2),
+            website: 'http://www.apple.com/accessibility/osx/voiceover'
+        }, {
+            name: 'ZoomText',
+            y: 2,
+            color: getColorPattern(3),
+            website: 'http://www.zoomtext.com/products/zoomtext-magnifierreader'
+        }, {
+            name: 'Other',
+            y: 4.4,
+            color: getColorPattern(4),
+            website: 'http://www.disabled-world.com/assistivedevices/computer/screen-readers.php'
+        }]
+    }],
+    responsive: {
+        rules: [{
+            condition: {
+                maxWidth: 500
+            },
+            chartOptions: {
+                plotOptions: {
+                    series: {
+                        dataLabels: {
+                            format: '<b>{point.name}</b>'
+                        }
+                    }
+                }
+            }
+        }]
+    }
+});
+
+Highcharts.chart('container4', {
+    chart: {
+        type: 'spline',
+        style: {
+            fontFamily: 'Roboto,Arial'
+        }
+    },
+
+    exporting: {
+        enabled: false
+    },
+
+    legend: {
+        enabled: false,
+        symbolWidth: 40
+    },
+
+    title: {
+        text: ''
+    },
+
+    yAxis: {
+        visible: false,
+        title: {
+            text: 'Percentage usage'
+        }
+    },
+
+    xAxis: {
+        visible: false,
+        title: {
+            text: 'Time'
+        },
+        accessibility: {
+            description: 'Time from December 2010 to September 2019'
+        },
+        categories: ['December 2010', 'May 2012', 'January 2014', 'July 2015', 'October 2017', 'September 2019']
+    },
+
+    tooltip: {
+        valueSuffix: '%',
+        enabled: false
```

Comment:
Tooltip disabled here but not for the others? Ditch the valueSuffix and add simple fake data as for the others?